src/main/resources/templates/index.html

```diff
@@ -2,65 +2,67 @@
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+            crossorigin="anonymous"></script>
 </head>
 <body>
-   <div th:replace="fragments/topnav :: navbar-dev"></div>
+<div th:replace="fragments/topnav :: navbar-dev"></div>
+<div class="dashboard-content">
+    <div class="panel-left">
+        <h3>My Recent Projects</h3>
+        <table class="table table-striped table-light">
+            <thead>
+            <tr>
+                <th scope="col">Project Id</th>
+                <th scope="col">Time</th>
+                <th scope="col">Client</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr th:each="employee:${employee}">
+                <td th:text="${employee.id}"></td>
+                <td>00:00</td>
+                <td>Client Name</td>
+                <td>
+                    <a role="button" th:href="@{projectdetails}" class="btn btn-secondary">Project Details</a>
+                </td>
+            </tr>
 
-    <div class="dashboard-content">
-        <div class="panel-left">
-            <h3>My Recent Projects</h3>
-            <table class="table table-striped table-light">
-                <thead>
-                <tr>
-                    <th scope="col">Project Id</th>
-                    <th scope="col">Time</th>
-                    <th scope="col">Client</th>
-                </tr>
-                </thead>
-                <tbody>
-
-                <tr th:each="employee:${employee}">
-                    <td th:text="${employee.id}"></td>
-                    <td>00:00</td>
-                    <td>Client Name</td>
-                    <td>
-                        <a role="button" th:href="@{projectdetails}" class="btn btn-secondary">Project Details</a>
-                    </td>
-                </tr>
-
-                </tbody>
-            </table>
-        </div>
-
-        <div class="panel-right">
-            <h4>Developer Time Logger</h4>
-            <form action="#" th:action="@{/logTime}" th:object="${employee}" method="GET">
-                <div class="form-group">
-                    <label for="clientInput">Client</label>
-                    <input type="text" class="form-control" id="clientInput" placeholder="Client">
-
-                </div>
-                <div class="form-group">
-                    <label for="projectInput">Project</label>
-                    <input type="text" class="form-control" id="projectInput" placeholder="Project">
-                </div>
-                <div class="form-group">
-                    <label for="hoursInput">Enter Time</label>
-                    <input type="text" class="form-control" id="hoursInput" placeholder="00:00">
-                </div>
-                <button name="submit" value="0" type="submit" class="btn btn-primary">Submit</button>
-            </form>
-        </div>
```

Comment:
This was simply a code reformatting using IntelliJ's built in Reformat Code option